doc/source/building/build_live_iso.rst

```diff
@@ -69,3 +70,60 @@ be tested with QEMU:
 The image is now complete and ready to use. See :ref:`iso_to_usb_stick` and
 :ref:`iso_as_file_to_usb_stick` for further information concerning
 deployment.
+
+.. _live_features:
+
+Decision for a live ISO technology
+----------------------------------
+
+The decision for the `overlay` or `dmsquash` dracut module depends on
```

Comment:
Probably here at the beginning we could state that `overlay` module is mostly conceived around an overlayfs deployment for write partitions and `dmsquash` is conceived around device-mapper snapshots.  I know you already noted that in the feature list, however this difference is what actually makes them a different creature.

Reply:
Actually I think the reason for dmsquash in the beginning was that there was only squashfs and the device mapper as the only possibility to map write actions to somewhere else. With the invention of the overlayfs kernel module things changed. But I agree from a concept perspective the dmsquash module was written as a generic device mapping technology whereas the kiwi overlay module had a smaller scope. I also added some information here and hope this does not confuse people :)